solutions/standard/variables.tf

```diff
@@ -315,3 +315,25 @@ variable "admin_pass_sm_secret_name" {
   description = "The name of a new elasticsearch administrator secret. If a prefix input variable is specified, the prefix is added to the name in the `<prefix>-<name>` format."
   default     = "elasticsearch-admin-password"
 }
+
+##############################################################
+# Kibana Configuration
+##############################################################
+
+variable "code_engine_project_name" {
```

Comment:
This should be called `existing_code_engine_project_name` as its not used when creating a new project